### DISCUSSION TITLE
Does Explorer Patcher Work After Replacing Explorer?

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#3012] [Q&A]
will Explorer Patcher still work if you replace the Windows 11 explorer with the Windows 10 one? When replacing the Windows 11 explorer with the Windows 10th one gives more of the Windows 10 features, like when you right on the taskbar. File Explorer returns to the Windows 10 one. But it obviously doesn't change the start menu to the Windows 10th one.

**Comment by Amrsatrio** [comment]
Not recommended at all. If you have multiple input languages, the Windows 10 explorer would crash continuously on 22H2.

**Reply by TimMarvelHero** [reply]
how could you add the Windows 10 start menu through command prompt then?
